Makefile

```diff
@@ -17,7 +17,7 @@ pylint:
 	pylint --rcfile .pylintrc geopy
 
 test:
-	nosetests --verbose --with-cover --cover-erase --cover-package=geopy test.geocoders.googlev3 test.geocoders.nominatim test.geocoders.photon test.geocoders.base test.geocoders.util
+	nosetests --verbose --nocapture --with-cover --cover-erase --cover-package=geopy test.geocoders.googlev3 test.geocoders.nominatim test.geocoders.photon test.geocoders.base test.geocoders.util test.geocoders.googleplaces
```

Comment:
nocapture allows print statements to be showed on the console. otherwise they are caught by nose. It was the only way I could easily debug.